src/ResultWriter/AnalysisWriter.cpp

```diff
@@ -53,6 +53,11 @@ void AnalysisWriter::printAnalysis(double simulationTime) {
      initialConditionType == seissol::initializer::parameters::InitializationType::PressureInjection) {
     return;
   }
+
+  if (initialConditionType == seissol::initializer::parameters::InitializationType::AcousticTravellingwithITM) {
```

Comment:
Do we need this If statement here? It looks a bit out of place to only do it for one initial condition

Reply:
Need? No. I wanted it to show the log info for this specific initial condition as it is quite specific as it is for only one particular case with ITM for which we did our convergence analysis. So, I think it is nice to have so that anyone in the future who wants to do any convergence analysis with ITM can see in the log when it is chosen. But it can be removed.

Reply:
There should be a log entry for the initial condition at the beginning of the simulation. What you could add is to notify the user if the ITM is active (i.e. at the `ITMToggle` if in the `InitModel.cpp` file)